py/server/deephaven/parquet.py

```diff
@@ -76,30 +80,72 @@ def _build_parquet_instructions(
     if max_dictionary_size is not None:
         builder.setMaximumDictionarySize(max_dictionary_size)
 
-    builder.setIsLegacyParquet(is_legacy_parquet)
+    if is_legacy_parquet:
+        builder.setIsLegacyParquet(is_legacy_parquet)
 
     if target_page_size is not None:
         builder.setTargetPageSize(target_page_size)
 
-    builder.setIsRefreshing(is_refreshing)
+    if is_refreshing:
+        builder.setIsRefreshing(is_refreshing)
 
     return builder.build()
 
+def _j_table_definition(table_definition: Union[Dict[str, DType], List[Column], None]):
```

Comment:
@jmao-denver Should this go in `table.py` or some other location where it could be reused?